app/src/main/java/org/sopt/cgv/feature/seats/component/SeatSelectionStepperRow.kt

```diff
@@ -0,0 +1,45 @@
+package org.sopt.cgv.feature.seats.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import org.sopt.cgv.core.designsystem.theme.Black
+import org.sopt.cgv.core.designsystem.theme.CGVTheme
+
+@Composable
+fun StepperRow(
+    modifier: Modifier = Modifier,
+    viewModel: SeatSelectViewModel,
```

Comment:
```suggestion
     value: Int,
    onIncrease: () -> Unit,
    onDecrease: () -> Unit
```
이 코드에서도 ViewModel 직접 참조 대신 필요한 상태와 이벤트만 매개변수로 받도록 변경해주세요! 이렇게 바꾸면 컴포넌트가 특정 ViewModel에 종속되지 않아 다양한 상황에서 재사용이 가능할 거 같아요!

Reply:
또한 이 컴포넌트 함수명을 파일명과 일치시키면 조금 더 보기 쉬울 거 같아요!